App/Redux/UIRedux.ts

```diff
@@ -86,10 +87,9 @@ export type UIState = {
     readonly photoId?: string
     readonly threadId?: string
   },
-  readonly sharingPhoto: {
-    readonly active: boolean,
-    readonly hash?: string,
-    readonly selectedThreads: ReadonlyMap<string, boolean>,
+  readonly sharingPhoto?: {
```

Comment:
Centralized the state of picking a photo to share, adding a comment, etc. Makes it easier to not have to pass state between screens in the nav props. Also, this state works for shares from the Wallet or from the camera roll (note `image?: SharedImage | string`).

Reply:
very cool